src/app/filter/filter.service.spec.ts

```diff
@@ -13,4 +13,37 @@ describe('FilterService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('should have values$ Observable', () => {
+    expect(service.values$).toBeDefined();
```

Comment:
you could move this below and making a comparison with the other expectation, like values are not equal to currentPage 1 and entriesPerPage 20 (?)

check the differences before the subscribe and after it (if it changes), or just merge the two test case